src/applications/ivc-champva/10-10D/pages/ApplicantMedicareStatusContinuedPage.jsx

```diff
@@ -54,38 +41,19 @@ export function generateOptions({ data, pagePerItemIndex }) {
 export function ApplicantMedicareStatusContinuedPage(props) {
   const newProps = {
     ...props,
-    keyname: KEYNAME,
-    primary: PRIMARY,
-    secondary: SECONDARY,
+    ...PROPERTY_NAMES,
     genOp: generateOptions,
   };
   return ApplicantRelationshipPage(newProps);
 }
 export function ApplicantMedicareStatusContinuedReviewPage(props) {
   const newProps = {
     ...props,
-    keyname: KEYNAME,
-    primary: PRIMARY,
-    secondary: SECONDARY,
+    ...PROPERTY_NAMES,
     genOp: generateOptions,
   };
   return ApplicantRelationshipReviewPage(newProps);
 }
 
-ApplicantMedicareStatusContinuedReviewPage.propTypes = {
-  data: PropTypes.object,
-  editPage: PropTypes.func,
-  pagePerItemIndex: PropTypes.number,
-  props: PropTypes.object,
-  title: PropTypes.func,
-};
-
-ApplicantMedicareStatusContinuedPage.propTypes = {
-  data: PropTypes.object,
-  goBack: PropTypes.func,
-  goForward: PropTypes.func,
-  pagePerItemIndex: PropTypes.string,
-  setFormData: PropTypes.func,
-  updatePage: PropTypes.func,
-  onReviewPage: PropTypes.bool,
-};
+ApplicantMedicareStatusContinuedReviewPage.propTypes = reviewPageProps;
+ApplicantMedicareStatusContinuedPage.propTypes = pageProps;
```

Comment:
Replaced with the common page props that were added in the `constants` file. All the pages that use the `ApplicantRelationshipPage` custom page base can just use these same props types